doc/api/process.md

```diff
@@ -706,14 +706,15 @@ console.log(process.env.test);
 // => 1
 ```
 
-## process.emitWarning(warning[, name][, ctor])
+## process.emitWarning(warning[, type[, code]][, ctor])
```

Comment:
The API *already* supports two call signatures. The first argument can be passed in as a pre-constructed `Error` object with all of the relevant properties set:

```js
const warning = new Error('This is a warning');
warning.name = 'CustomWarning';
warning.code = 'ABC123';
process.emitWarning(warning);
```

is equivalent to:

```js
process.emitWarning('This is a warning', 'CustomWarning', 'ABC123');
```

Having an options object when we can already pass in an `Error` object would be rather pointless, IMO